.circleci/config.yml

```diff
@@ -29,11 +29,25 @@ jobs:
                 command: |
                     cd docker/emptydrops/npz2rds/test/
                     ./test.sh
+
+    test_snap_atac:
+        docker:
+            - image: circleci/python:3.6.1
+        steps:
+            - checkout
+            - run:
+                  command: |
+                      python3 -m venv venv
+                      . venv/bin/activate
+                      pip install requests
+                      python test/run_portability_test.py --test-directories test/snap-atac/pr/
```

Comment:
instead of using portability test, use the pr trigger test script. Once you update your branch from master, you'll see an example of how to do that in this script.